projects/storefrontlib/shared/components/media/media.component.ts

```diff
@@ -61,6 +66,30 @@ export class MediaComponent implements OnChanges {
    */
   @Input() loading: ImageLoadingStrategy | null = this.loadingStrategy;
 
+  @Input() useImgElement: boolean = false;
```

Comment:
```suggestion
  @Input() elementType: 'picture' | 'img' = 'picture'
```
`query:` What do you think about changing the type of this input from boolean to enum? Seems to be a better choice semantic-wise and such an approach allows us to adjust the type in case new elements for handling images appear in the future.
TBH. The best solution for similar cases is content projection, but I'm not sure whether it will help here.

Reply:
Make sens, changed